File: Chuongtrinh/cronjob/model/document.go
package model

type Document struct {
	ID           int64     `gorm:"primary_key;column:ID"`
	Ten          string    `gorm:"column:Ten"`
	Nguoisohuu   string    `gorm:"column:Nguoisohuu"`
	Image        string    `gorm:"column:Image"`
	MaLop        int64     `gorm:"column:MaLop"`
	LuotTaiXuong int64     `gorm:"column:LuotTaiXuong"`
	Luotxem      int64     `gorm:"column:Luotxem"`
	IDLibrary    int64     `gorm:"column:IDLibrary"`
	LopHoc       *Lophoc   `gorm:"ForeignKey:ID;references:MaLop;"`
	TaiKhoan     *Taikhoan `gorm:"ForeignKey:TenDangNhap;references:Nguoisohuu;"`
	Library      *Library  `gorm:"ForeignKey:ID;references:IDLibrary;"`
}

File: Chuongtrinh/DOANTOTNGHIEP/DOANTOTNGHIEP/Models/document.cs
namespace DOANTOTNGHIEP.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("document")]
    public partial class document
    {
        public long ID { get; set; }

        public string Ten { get; set; }

        [StringLength(20)]
        public string Nguoisohuu { get; set; }


        public long? MaLop { get; set; }

        public int? LuotTaiXuong { get; set; }

        public int? Luotxem { get; set; }

        public long? IDLibrary { get; set; }

        public virtual LopHoc LopHoc { get; set; }

        public virtual TaiKhoan TaiKhoan { get; set; }

        public virtual Library Library { get; set; }
    }
}
